price_parser/parser.py

```diff
@@ -92,11 +105,16 @@ def or_regex(symbols: List[str]) -> Pattern:
 DOLLAR_CODES = [k for k in CURRENCY_CODES if k.endswith('D')]
 DOLLAR_REGEXES = [
     r"""
-    \b{}   # code like NZD
-    (?:[^\w]|$)  # not a letter
+    (\b{}   # code like NZD
+    (?:[^\w]|$))  # not a letter
     """.format(k) for k in DOLLAR_CODES
 ]
 
+OTHER_PARTICULAR_REGEXES = [
+    # HT is the French abbreviation for "Hors Tax" (tax not added to the price)
+    # and it may appear after € currency symbol
+    r"(€)HT+?",
```

Comment:
Can’t the same be achieved by adding `'€HT'` to `SAFE_CURRENCY_SYMBOLS`?

Reply:
`€HT` is not a currency symbol (only `€` is) , so it won't work just including there. It is a more specific case.